test/aria/html/select/onchange/DataModelOnChangeRobotTestCase.js

```diff
@@ -36,14 +36,19 @@ Aria.classDefinition({
 
             this.assertEquals(selectWidget.selectedIndex, 0, "The selected Index should be %2 but was %1");
 
-            this.synEvent.execute([["click", selectWidget], ["type", null, "[down][down][enter]\t"]], {
+            this.synEvent.execute([
+                ["click", selectWidget],
+                ["waitFocus", selectWidget],
+                ["type", null, "[down][down][enter]\t"],
+                ["pause", 1000]
```

Comment:
@benouat Thank you for your comment. I have changed the code to use `waitFor` instead.